File: Selenium/Project/Activity4.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Firefox() as driver:
    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/jobs")

    # Print the second header of the page
    second_heading = driver.find_element(By.XPATH, "//h2[contains(text(), 'Quia')]")
    print("Second heading is: ", second_heading.text)
    

File: Selenium/Project/Activity4.java
package project;

import static org.testng.Assert.assertEquals;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity4 {
	//Declare webdriver
	WebDriver driver;
	
	//Set up method
	@BeforeClass
	  public void setUp() {
		   {
		        // Initialize webdriver
		        driver = new FirefoxDriver();
		        //Open the page
		        driver.get("https://alchemy.hguy.co/jobs");
		  }
	  }
		
  @Test(priority = 1)
  public void pageHeadingTest() {
	  //Get the page subheading and verify it

	  String secondHeaderText = driver.findElement(By.xpath("//h2[contains(text(), 'Quia')]")).getText();
	  System.out.println(secondHeaderText);
	  assertEquals(secondHeaderText,"Quia quis non");
	  
  }
  
  
  @AfterClass
  public void tearDown() {
      // Close the browser
      driver.quit();
  }
  
}
